src/test/resources/testconfig.properties

```diff
@@ -4,5 +4,5 @@
 Verify.matlab.invokes.positive = MATLAB is invoking positive tests
```

Comment:
What do you mean by positive test?

Reply:
These are tests which will not fail and are straight forward scenarios.